CMakeLists.txt

```diff
@@ -1042,6 +1042,8 @@ if(BUILD_CURL_EXE)
   add_subdirectory(src)
 endif()
 if(BUILD_CURL_TESTS)
+  include(CTest)
+  enable_testing()
```

Comment:
The `CTest` module calls `enable_testing` for us so we don't need to call it here.